jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -1,12 +1,21 @@
 pipeline {
-    agent none
+    agent any
 
     parameters {
         // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker02.web.prod.ext.phx2.redhat.com-44775-1580379079663-8:893046:0:0:1&is_raw=true&size=extra-large
         string defaultValue: '', description: 'This will be populated by the CI trigger', name: 'CI_MESSAGE', trim: true
     }
-
     stages {
+        stage('Install yq') {
```

Comment:
Maybe make this stage more generic `stage('Ensure Dependencies')`, we will likely remove the yq installation, but i guess we could have a stage to check stuff before continuing.